packages/web-components/@groupby/elements-products/test/unit/products.test.ts

```diff
@@ -55,6 +60,31 @@ describe('Products Base Component', () => {
       expect(setAttribute).to.not.be.called;
     });
   });
+
+  describe('requestInitialData', () => {
+    it('should dispatch a cache request event', () => {
+      const dispatchElementsEvent = stub(component, 'dispatchElementsEvent');
+      const name = 'event';
+      const group = 'group';
+      const returnEvent = 'event-return';
+
+      component.requestInitialData(name, group, returnEvent);
+
+      expect(dispatchElementsEvent).to.have.been.calledWith(CACHE_REQUEST, { name, group, returnEvent });
```

Comment:
I think we've been using the simple `to.be.calledWith` elsewhere.

```suggestion
      expect(dispatchElementsEvent).to.be.calledWith(CACHE_REQUEST, { name, group, returnEvent });
```